keras_cv/models/convmixer.py

```diff
@@ -0,0 +1,355 @@
+    pooling=None,
+    classifier_activation="softmax",
+    name="ConvMixer_1536_24",
+    **kwargs,
+):
+    """Instantiates the ConvMixer_1536_24 architecture."""
+
+    return ConvMixer(
+        dim=MODEL_CONFIGS["ConvMixer_1536_24"]["dim"],
+        depth=MODEL_CONFIGS["ConvMixer_1536_24"]["depth"],
+        patch_size=MODEL_CONFIGS["ConvMixer_1536_24"]["patch_size"],
+        kernel_size=MODEL_CONFIGS["ConvMixer_1536_24"]["kernel_size"],
+        include_rescaling=include_rescaling,
+        include_top=include_top,
+        name=name,
+        weights=weights,
+        input_shape=input_shape,
+        input_tensor=input_tensor,
+        pooling=pooling,
+        classes=classes,
+        classifier_activation=classifier_activation,
+        **kwargs,
+    )
+
+
+def ConvMixer_768_32(
+    include_rescaling,
+    include_top,
+    classes=None,
+    weights=None,
+    input_shape=(None, None, 3),
+    input_tensor=None,
+    pooling=None,
+    classifier_activation="softmax",
+    name="ConvMixer_768_32",
+    **kwargs,
+):
+    """Instantiates the ConvMixer_768_32 architecture."""
+
+    return ConvMixer(
+        dim=MODEL_CONFIGS["ConvMixer_768_32"]["dim"],
+        depth=MODEL_CONFIGS["ConvMixer_768_32"]["depth"],
+        patch_size=MODEL_CONFIGS["ConvMixer_768_32"]["patch_size"],
+        kernel_size=MODEL_CONFIGS["ConvMixer_768_32"]["kernel_size"],
+        include_rescaling=include_rescaling,
+        include_top=include_top,
+        name=name,
+        weights=weights,
+        input_shape=input_shape,
+        input_tensor=input_tensor,
+        pooling=pooling,
+        classes=classes,
+        classifier_activation=classifier_activation,
+        **kwargs,
+    )
+
+
+def ConvMixer_1024_16(
+    include_rescaling,
+    include_top,
+    classes=None,
+    weights=None,
+    input_shape=(None, None, 3),
+    input_tensor=None,
+    pooling=None,
+    classifier_activation="softmax",
+    name="ConvMixer_1024_16",
+    **kwargs,
+):
+    """Instantiates the ConvMixer_1024_16 architecture."""
+
+    return ConvMixer(
+        dim=MODEL_CONFIGS["ConvMixer_1024_16"]["dim"],
+        depth=MODEL_CONFIGS["ConvMixer_1024_16"]["depth"],
+        patch_size=MODEL_CONFIGS["ConvMixer_1024_16"]["patch_size"],
+        kernel_size=MODEL_CONFIGS["ConvMixer_1024_16"]["kernel_size"],
+        include_rescaling=include_rescaling,
+        include_top=include_top,
+        name=name,
+        weights=weights,
+        input_shape=input_shape,
+        input_tensor=input_tensor,
+        pooling=pooling,
+        classes=classes,
+        classifier_activation=classifier_activation,
+        **kwargs,
+    )
+
+
+def ConvMixer_512_16(
+    include_rescaling,
+    include_top,
+    classes=None,
+    weights=None,
+    input_shape=(None, None, 3),
+    input_tensor=None,
+    pooling=None,
+    classifier_activation="softmax",
+    name="ConvMixer_512_16",
+    **kwargs,
+):
+    """Instantiates the ConvMixer_512_16 architecture."""
```

Comment:
It looks like these docstrings for specific versions of the architecture are missing the details from the docstring for the base model method.

Please add complete docstrings for these methods (the approach using `setattr` in e.g. DenseNet is a good example of how to do this without a ton of duplicate code)

Reply:
Done!